bundle/src/main/java/com/cognifide/cq/cqsm/core/executors/ModifyExecutor.java

```diff
@@ -62,61 +53,29 @@
 	@Reference
 	private EventListener eventListener;
 
-	@Reference
-	private ScriptManager scriptManager;
-
-	@Reference
-	private ScriptFinder scriptFinder;
-
-	@Reference
-	private ResourceResolverFactory resolverFactory;
-
 	@Activate
-	private synchronized void activate(ComponentContext ctx) {
-		SlingHelper.operateTraced(resolverFactory, new OperateCallback() {
-			@Override
-			public void operate(ResourceResolver resolver) throws Exception {
-				runModified(resolver);
-			}
-		});
+	private synchronized void activate() {
+		SlingHelper.operateTraced(resolverFactory, this::runModified);
 	}
 
 	private void runModified(ResourceResolver resolver) throws PersistenceException {
 		final List<Script> scripts = scriptFinder.findAll(filterOnModify(resolver), resolver);
 
-		if (scripts.size() > 0) {
-			LOG.info("Executor will try to run following scripts: {}", scripts.size());
-			LOG.info(MessagingUtils.describeScripts(scripts));
-
+		if (scripts.isEmpty()) {
+			if(LOG.isInfoEnabled()) {

```

Comment:
Good example of proper usage this method. It should be used only when computation of message can be expensive.